other_procore_apis/bim_web_viewer.md

```diff
@@ -15,7 +15,9 @@ section_title: Other Procore APIs
 
 <p class="heading-link-container"><a class="heading-link" href="#installation-from-npm-recommended"></a></p>
 
-The Procore BIM Web Viewer is available as a module on `npm` as [`@procore/bim-webviewer-sdk`](https://www.npmjs.com/package/@procore/bim-webviewer-sdk).
+![](https://img.shields.io/npm/v/@procore/bim-webviewer-sdk?color=%23f47e42)
+
+The Procore BIM Web Viewer is available as a module on `npm` as [`@procore/bim-webviewer-sdk`](https://www.npmjs.com/package/@procore/bim-webviewer-sdk). The latest version is 
```

Comment:
The suspense is killing me, what is the latest version??

Reply:
LOL whoops 😅